### PR TITLE
Add support multipart data fields

### PR DESCRIPTION
**Pull request by intech**
Fix empty options
Add multipart field data in ctx.params
Add tests

### CODE REVIEW

File: test/integration/index.spec.js [hunk 1]
```diff
@@ -3268,6 +3335,31 @@ describe("Test file uploading", () => {
...
 
+	it("should send data field with multipart", () => {
+		return request(server)
+			.post("/upload")
+			.attach("myFile", assetsDir + "logo.png")
+			.field("name", "moleculer")
+			.then(res => {
+				expect(res.statusCode).toBe(200);
+				expect(res.headers["content-type"]).toBe("application/json; charset=utf-8");
+				expect(res.body).toEqual('moleculer');
```
Comment (icebob) [inline test/integration/index.spec.js]: should check that file is uploaded successfully, as well.

Comment (intech) [reply test/integration/index.spec.js]: Done!

**Comment by intech** [comment]
Problem... one moment

**Comment by icebob** [comment]
I'm considering that which is better. Push the fields to the `ctx.params` or `ctx.meta`. Because if you push to `ctx.params`, it won't be transferred to the services because ctx.params is the stream of the file. But `ctx.meta` reaches the target services, as well. What do you think?

**Comment by intech** [comment]
I think it is possible in meta, the main thing we will discribe it in documentation

**Comment by icebob** [comment]
Ok. But in this case put the fields under `ctx.meta.$multipart`. Because someone sends a multipart request with `user` field, they wouldn't overwrite the `ctx.meta` fields....like the authenticated user.

**Comment by intech** [comment]
I agree, a good decision. I'll do it now

**Comment by icebob** [comment]
Thanks, good job 👍

**Pull request merged**